crypto/ec/ec_lib.c

```diff
@@ -66,13 +66,13 @@ EC_GROUP *EC_GROUP_new(const EC_METHOD *meth)
 void EC_pre_comp_free(EC_GROUP *group)
 {
     switch (group->pre_comp_type) {
-    default:
+    case PCT_none:
         break;
-#ifdef ECP_NISTZ256_REFERENCE_IMPLEMENTATION
     case PCT_nistz256:
+#ifdef ECP_NISTZ256_REFERENCE_IMPLEMENTATION
```

Comment:
But it ruined mine :-) I mean the whole point is that one should replace ECP_NISTZ256_REFERENCE_IMPLEMENTATION with ECP_NISTZ256_ASM in *whole* ec_lib.c. There are two occurrences...